Dynamic-Landing-Page/script.js

```diff
@@ -0,0 +1,82 @@
+const wrapper = document.querySelector(".wrapper");
+const time = document.querySelector(".time");
+const greeting = document.querySelector(".greeting");
+const name = document.querySelector(".name");
+const Focus = document.querySelector(".focus");
+function ShowTime() {
+  let today = new Date();
+  let hour = today.getHours();
+  let min = today.getMinutes();
+  let second = today.getSeconds();
+  let amPm = hour >= 12 ? "PM" : "AM";
+  hour = hour % 12 || 12;
+  time.innerHTML = `${hour}:${addZero(min)}:${addZero(second)} ${amPm}`;
+  setTimeout(ShowTime, 1000);
+}
+function addZero(numb) {
+  return (numb < 10 ? "0" : "") + numb;
+}
+function setGreet() {
+  let today = new Date();
+  let hour = today.getHours();
+  if (hour < 12) {
+    wrapper.style.backgroundImage = "url(img/morning.jpg)";
+    greeting.textContent = "Good Morning, ";
+  } else if (hour < 18) {
+    wrapper.style.backgroundImage = "url(img/aftenoon.jpg)";
+    greeting.textContent = "Good Morning, ";
```

Comment:
If I understand correctly the greeting shouldn't be all time - "good morning"

Reply:
yes, fixed